game/templates/game/landing.html

```
@@ -103,11 +103,41 @@
       </div>
       <div class="nav-right">
         {% if user.is_authenticated %}
-        <a href="{% url 'index' %}" class="btn-signin">Dashboard</a>
-        <form method="post" action="{% url 'logout' %}" style="display: inline">
-          {% csrf_token %}
-          <button type="submit" class="btn-register">Log Out</button>
-        </form>
+        <div class="profile-dropdown">
+          <button class="profile-btn">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)